charts/localstack/templates/configmap.yaml

```diff
@@ -10,5 +10,6 @@ metadata:
     {{- include "localstack.annotations" . | nindent 4 }}
 data:
   {{ template "localstack.fullname" . }}-init-scripts-config.sh: |
+    !#/bin/sh
```

Comment:
Unfortunately, this change would prevent users from using anything else than `/bin/sh` to execute their scripts (like bash, or python).
But I guess there should be a more elaborate documentation on the necessity of a shebang line...
Would it be possible to contribute a documentation change instead of this code change?